packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -447,14 +464,43 @@ class TextSelectionOverlay {
     _updateTextSelectionOverlayVisibilities();
   }
 
+  void _disposeHandlesDismissalTimer() {
+    _handlesDismissalTimer?.cancel();
+    _handlesDismissalTimer = null;
+  }
+
+  // The duration for auto-dismissal of handles on Android.
+  static const Duration _androidHandlesDismissalDuration = Duration(seconds: 4);
```

Comment:
Could you add a sentence explaining where you got the 4 seconds from? Like "Measured on a physical Pixel 9 running Android 16." or whatever.